src/renderer/entities/notification/model/notification-model.ts

```diff
@@ -1,13 +1,39 @@
 import { createEffect, createEvent, createStore, sample } from 'effector';
+import { delay } from 'patronum';
 
 import { storageService } from '@/shared/api/storage';
-import { type NoID, type Notification } from '@/shared/core';
+import { type NoID, type Notification, NotificationType } from '@/shared/core';
 import { merge } from '@/shared/lib/utils';
 
+const BATCH_DELAY = 1000;
+
+export type NotificationToast = Pick<Notification, 'type' | 'status' | 'title' | 'description' | 'deepLink'> & {
+  titleParams?: { count?: number };
+};
+
+const getBatchedTitle = (type: NotificationType): string => {
```

Comment:
I think it's a bad idea, notification model shouldn't know about different notification types